llama/vendoring.env

```diff
@@ -0,0 +1 @@
+LLAMACPP_BASE_COMMIT=3f1ae2e32cde00c39b96be6d01c2997c29bae555
```

Comment:
Is there an ergonomic way to make this a file that has the hash (vs a .env file, which took me off guard for a minute as those aren't usually committed)

Reply:
`vendoring.def` or just `vendoring` maybe?